common/emu_constants.h

```diff
@@ -350,6 +350,7 @@ namespace EQ
 		};
 
 		enum EmoteTypes : uint8 {
+			Say,
```

Comment:
So putting this here is a big no-no. 

These are stored in the database in the order they are in this enum and you just threw `Say` (0) where `Emote` (0) is otherwise

This is stored as type and emote is (0). If you're going to add say add it to the end 

![image](https://github.com/EQEmu/Server/assets/3319450/c701ee69-89ba-4ddb-8a0b-f771bc64250a)

Reply:
![image](https://github.com/EQEmu/Server/assets/89047260/ef6e97bb-d540-4bf7-82c6-b71fb1426f73)
`Say` is `0`. The `enum` was wrong.

Reply:
We only use types `0` to `3` and we didn't have `0` in the enum and we had `Emote`/`Shout`/`Proximity` as `0`/`1`/`2` incorrectly when they are `1`/`2`/`3`.

Reply:
You're right, it was there all along. It looked like you added something to the source of truth enum for types